### ISSUE TITLE
Trying to compile on El Capitan, getting git error and MHTD variable error?

### ISSUE THREAD

**Issue by maxiedaniels** [fireice-uk/xmr-stak#400]
I can't find a solution anywhere, can someone help me?

My command and output:

```
$ cmake -DMICROHTTPD_ENABLE=OFF -DOpenSSL_ENABLE=OFF -DCUDA_ENABLE=OFF .
-- Set miner currency to 'monero' and 'aeon'
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
MHTD
    linked by target "xmr-stak-c" in directory /Users/myuser/Downloads/xmr-stak-master

-- Configuring incomplete, errors occurred!
See also "/Users/myuser/Downloads/xmr-stak-master/CMakeFiles/CMakeOutput.log".
See also "/Users/myuser/Downloads/xmr-stak-master/CMakeFiles/CMakeError.log".
```

**Comment by parkerjm** [comment]
I had a similar error. I installed the `libmicrohttpd` package via `brew` and stopped getting it.  You might have to blow away all of the files `cmake` generates first.

**Comment by fireice-uk** [comment]
This should be resolved in 2.1, please reopen if not
